src/translations/en.json

```diff
@@ -145,7 +147,8 @@
     "download.message.studies": "studies",
     "download.message.others": "others",
     "download.error": "An error occurred while downloading element {elementName}",
-    "MODIFICATION": "Composite modification",
+    "MODIFICATION": "Network modification",
```

Comment:
you need to modify this ?

Reply:
No I don't really need it. 

But this is a bit strange to use "Composite modification" for all the network modifications. Even if -for now- we can only visualize composite network modification in the explorer.

Reply:
I think it's ok but you can check with PO.
And you can change the key with "networkModification"

Reply:
I reverted those changes.